coral/src/domain/team/team-api.ts

```diff
@@ -2,10 +2,14 @@ import api from "src/services/api";
 import { transformTeamNamesGetResponse } from "src/domain/team/team-transformer";
 import { KlawApiResponse } from "types/utils";
 
-const getTeams = () => {
+const getTeamNames = () => {
```

Comment:
couldn't we use `getAllTeamsSU` only? If we can confirm with backend that `getAllTeamsSUOnlyOnly` returns the names only from the `TeamModel` we could do that. 

So e.g. BrowseTopics (where currently SUOnly is used) could use the same cache as the approvals tables. 

Ugh you know what, I'll add a follow up issue for that one! Because it also depends on how we want to config our caching etc. 

(https://github.com/aiven/klaw/issues/682)

Reply:
You can resolve this, I'm just leaving it open so you see it :D

Reply:
Haha yes, the `getAllTeamsSUOnlyOnly` seems very specific, and probably redundant. But probably could be handled later, for sure.